torchrl/data/replay_buffers/storages.py

```diff
@@ -29,6 +29,9 @@ class Storage:
 
     def __init__(self, max_size: int) -> None:
         self.max_size = int(max_size)
+        # Prototype feature. RBs that use a given instance of Storage should add
+        # themselves to this set.
+        self.attached_entities = set()
```

Comment:
do we want this to be public? Is any other object accessing that?

Reply:
Nope. I can rename it.